starknet_py/utils/data_transformer/universal_deployer_serializer.py

```diff
@@ -0,0 +1,39 @@
+from starkware.starknet.public.abi_structs import identifier_manager_from_abi
+
+from starknet_py.utils.data_transformer.data_transformer import CairoSerializer
+
+universal_deployer_abi = [
```

Comment:
ABI correctness should be checked, in conjunction with the addresses being correct (we should check if the contract is there, and has expected ABI). 
It can be either performed in a separate action which could be executed daily, or in test suite.